utils/remote.cc

```diff
@@ -118,7 +118,17 @@ static std::string etaToString(int64_t eta)
         return fmt::format(FMT_STRING("{:d} hrs"), eta / (60 * 60));
     }
 
-    return fmt::format(FMT_STRING("{:d} days"), eta / (60 * 60 * 24));
+    if (eta < (60 * 60 * 24 * 1000))
+    {
+        return fmt::format(FMT_STRING("{:d} days"), eta / (60 * 60 * 24));
+    }
+
+    if (eta < (60 * 60 * 24 * 365 * 100LL))
+    {
+        return fmt::format(FMT_STRING("{:d} years"), eta / (60 * 60 * 24 * 365));
+    }
+
+    return "∞";
```

Comment:
I'd remove the `if` and the inf symbol, and just always return years if months isn't enough

Reply:
> Completely different approach might be implementing new formatting with column sizes determined 'on-the-fly'.

I think there are some user scripts that depend on the width. It might be better for a one-time change that widens the eta_str width by a character or two so that user scripts can get updated and move on? (Simpler code change for us, too :smile_cat:)